apps/web/test/lib/slots.test.ts

```diff
@@ -26,6 +26,7 @@ it("can fit 24 hourly slots for an empty day", async () => {
           endTime: MINUTES_DAY_END,
         },
       ],
+      eventLength: 60,
```

Comment:
We can add a test with different eventLengths just to try this

Reply:
Added a test with different frequency and eventLength to showcase where they vary 😄